browser/brave_shields/brave_timestamp_rounding_browsertest.cc

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/test/scoped_feature_list.h"
+#include "base/test/task_environment.h"
+#include "base/time/time.h"
+#include "brave/components/brave_shields/common/features.h"
```

Comment:
I don't think this include is necessary.

Reply:
fixed. Thank you for the review, @goodov !